test/factory/common_factory_tests.dart

```diff
@@ -38,65 +37,22 @@ void commonFactoryTests(ReactComponentFactoryProxy factory) {
 
       const firstGeneralCaseVariadicChildCount = 2;
       const maxSupportedVariadicChildCount = 40;
-      for (var i = firstGeneralCaseVariadicChildCount;
-          i < maxSupportedVariadicChildCount;
-          i++) {
+      for (var i = firstGeneralCaseVariadicChildCount; i < maxSupportedVariadicChildCount; i++) {
         final childrenCount = i;
 
         test('$childrenCount', () {
-          final expectedChildren =
-              new List.generate(childrenCount, (i) => i + 1);
+          final expectedChildren = new List.generate(childrenCount, (i) => i + 1);
           final arguments = <dynamic>[{}]..add(expectedChildren);
           final instance = Function.apply(factory, arguments);
           expect(getChildren(instance), expectedChildren);
         });
       }
 
       test('$maxSupportedVariadicChildCount (and passes static analysis)', () {
-        final instance = factory({},
-            1,
-            2,
-            3,
-            4,
-            5,
-            6,
-            7,
-            8,
-            9,
-            10,
-            11,
-            12,
-            13,
-            14,
-            15,
-            16,
-            17,
-            18,
-            19,
-            20,
-            21,
-            22,
-            23,
-            24,
-            25,
-            26,
-            27,
-            28,
-            29,
-            30,
-            31,
-            32,
-            33,
-            34,
-            35,
-            36,
-            37,
-            38,
-            39,
-            40);
+        final instance = factory({}, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23,
+            24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40);
```

Comment:
MY 👏 POINT 👏 EXACLY 👏

Reply:
You mean you would have wanted this to be 40 different lines? I think two is better 😛

Reply:
You've obviously never seen your stats in GitPrime then :P